src/HandlerModulePlugin.h

```diff
@@ -17,8 +17,10 @@ class InputEventProxy;
 
 class HandlerModulePlugin : public AbstractPlugin
 {
-
 public:
+    static geo::MaxmindDB *asn;
```

Comment:
can/should we make these `const`? default to `nullptr`?

Reply:
it cannot be const because we should change it reference/value on runtime.


The default is a valid reference
HandlerModulePlugin.cpp 
```
geo::MaxmindDB *HandlerModulePlugin::asn = &geo::GeoASN();
geo::MaxmindDB *HandlerModulePlugin::city = &geo::GeoIP();
```

For static module this reference is enough... but for cutom runtime this reference exists but will be always not enabled DBs.. then when we init the custom handler plugin module we pass the correct loaded reference;

Reply:
So asn and city are never nullptr